apps/vr-tests/src/stories/ButtonNext.stories.tsx

```diff
@@ -131,8 +142,13 @@ storiesOf('Button Next - With icon before content', module)
       Hello, world
     </Button>
   ))
-  .addStory('Primary Disabled', () => (
-    <Button primary disabled icon="X">
+  .addStory('Transparent', () => (
+    <Button transparent icon="X">
+      Hello, world
+    </Button>
+  ))
+  .addStory('Transparent Disabled', () => (
```

Comment:
let's also add a test for both `transparent` and `ghost` are set (which is what user needs to do if they need `ActionButton`)